api-gateway/src/infrastructure/http/subscription.service.ts

```diff
@@ -60,7 +60,7 @@ export class SubscriptionServiceImpl implements SubscriptionService {
   async unsubscribe(email: string) {
     try {
       const response = await firstValueFrom(
-        this.httpService.post<void>(
+        this.httpService.delete<void>(
           `${this.subscriptionUrl}/subscribe/` + email,
```

Comment:
I might have missed this in the previous PR, but /subscribe/ endpoint is responsible for subscription and unsubscribing?

Reply:
Yes)) It handles subscriptions and unsubscriptions, differing only in the HTTP method.